lms/templates/edraak_university/university_id_success.html

```diff
@@ -40,5 +42,13 @@ <h1 class="valid text-center">${_("Your Student University ID Has Been Saved")}<
                     <strong>${_('View the Course Content')}</strong>
                 </a>
             </p>
+        % else:
+        <h1 class="valid text-center">${_("Your Settings Have Been Saved")}</h1>
+        <p class="text-center">
+            <a href="${reverse('edraak_university_id_staff', kwargs={'course_id': course.id})}">
+                <strong>${_('View the Students List')}</strong>
+            </a>
+        </p>
+        % endif
```

Comment:
using a separate view to implement success unless there was a way using context data @Salomari1987 @devalih